CareKitUI/CareKitUITests/Task/TestSliderTaskView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  TestSliderTaskView.swift
+//
+//
+//  Created by Dylan Li on 7/27/20.
+//  Copyright © 2020 NetReconLab. All rights reserved.
+//
+
+import CareKitUI
+import SwiftUI
+import XCTest
+
+class TestSliderTaskView: XCTestCase {
+    
+    @State var value: CGFloat = 0
```

Comment:
`@State` should only be used in a SwiftUI view. You can test the slider using a mock slider view, passing in a constant for the `value`:

```swift
struct SliderView: View {

    @Binding
    var value: CGFloat
}

struct MockSliderView {

    let value: CGFloat

    var body: some View {
        SliderView(value: .constant(value))
    }
}
```